pkg/controllermanager/controller/shoot/shoot_control_reconcile.go

```diff
@@ -76,6 +76,7 @@ func (c *Controller) runReconcileShootFlow(o *operation.Operation, operationType
 		managedInternalDNS        = o.Garden.InternalDomain != nil && o.Garden.InternalDomain.Provider != gardenv1beta1.DNSUnmanaged
 		creationPhase             = operationType == gardencorev1alpha1.LastOperationTypeCreate
 		requireKube2IAMDeployment = creationPhase || controllerutils.HasTask(o.Shoot.Info.Annotations, common.ShootTaskDeployKube2IAMResource)
+		allowBackup               = (o.Seed.CloudProfile.Spec.Backup != nil)
```

Comment:
Help me, I cannot recall exactly. Originally, we tried to achieve that the `Seed` does no longer reference a `CloudProfile` anymore. We cannot get rid of it because of the new backup infra solution you proposed, right?

Reply:
yes.

Reply:
See above question - how will this look like after GEP-4 is implemented?